scripts/suite.py

```diff
@@ -35,6 +35,11 @@
                               If both -S and -c are supplied, -S wins, and
                               there is no validation that sha1 is contained
                               in branch
+  -n, --newest                Search for the newest revision built on all
+                              required distro/versions, starting from
+                              either --ceph or --sha1.
+  --newest-limit <backtrack>  backtrack is the number of commits to try
+                              in --newest mode  [default: 10]
```

Comment:
Are these separate because you want to be able to pass ``-n`` and not e.g. ``-n 10`` all the time?